.github/workflows/autobuild.yaml

```diff
@@ -142,28 +142,28 @@ jobs:
       - name: Check if tag exists
         uses: mukunku/tag-exists-action@v1.6.0
         id: check-tag
-        with: 
-          tag: "${{steps.get-version.outputs.version}}"
+        with:
+          tag: ${{ env.version }}
 
       - name: Create Release
         if: steps.check-tag.outputs.exists == 'false'
         uses: actions/create-release@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          tag_name: "${{steps.get-version.outputs.version}}"
-          release_name: OpenTaiko v${{steps.get-version.outputs.version}}
+          tag_name: ${{ env.version }}

```

Comment:
This causes the workflow to fail because it expands to nothing when the commit message contains no version number.